### ISSUE TITLE
Bulk-update indicator detection

### ISSUE THREAD

**Issue by misje** [LuatixHQ/opencti#7815]
## Use case

Being able to update the detection state on selected indicators.

## Current Workaround

None that I am aware of.

**Comment by romain-filigran** [comment]
Thank you for your suggestion. We'll be working on this soon. For information, it's possible to update the 'detection' field of indicators in playbook through the "Manipulate knowledge" component. Maybe a workaround

**Comment by misje** [comment]
Thank you for the suggestion, but as far as I understand, I can't use the playbook to manipulate existing data? It will only trigger on creation, updates and deletions.

**Comment by romain-filigran** [comment]
Yes, you're right, it will only work on new or updated indicators.

**Comment by romain-filigran** [comment]
I am closing this issue as it has been resolved previously.
